ref/Microsoft.Build.Framework/net/Microsoft.Build.Framework.cs

```diff
@@ -4,7 +4,9 @@ namespace Microsoft.Build.Framework
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public partial struct BuildEngineResult
     {
-        public BuildEngineResult(bool result, System.Collections.Generic.List<System.Collections.Generic.IDictionary<string, Microsoft.Build.Framework.ITaskItem[]>> targetOutputsPerProject) { throw null;}
+        private object _dummy;
```

Comment:
>Adding some obvious garbage to the source.

Is this the garbage you were referring to? If so that is by design see https://github.com/dotnet/corefx/issues/6185.